index.html

```diff
@@ -1562,36 +1562,55 @@ <h2>Accessibility considerations</h2>
+    </p> 
+    <p>
+      Through the manifest's <a><code>req_permissions</code></a> member, developers declare the requirements of a MiniApp to access privacy-related sensitive and high-level privileged resources. This member allows the platform to understand the specific restricted resources the MiniApp may request (<a data-link-for="MiniApp permission resource" data-link-type="dfn"><code>name</code></a> member), informing the user about the potential consequences and granting (or keeping blocked) access to these resources. Developers SHOULD use the <a><code>reason</code></a> member to state the clear and easy-to-understand rationale for requesting these advanced resources. 
+    </p>
+    <p>
+      User agents MUST manage the user's preferences, including the explicit permissions the user shall grant to use sensitive system's features (e.g., geolocation and device's sensors). The user agent SHOULD implement mechanisms to remember their preferences, and the end-user SHALL explicitly decide if this information is stored across sessions. No other information related to the manifest should persist across sessions.
+    </p>
+    <p>
+      As in Web applications, MiniApps can be installed using UI dialogs. In the installation process, the user agent SHOULD display clear information about the app (i.e., <a><code>app_id</code></a>, <a data-link-for="MiniApp manifest" data-link-type="dfn"><code>name</code></a>, <a><code>short_name</code></a>, <a>icons</a>, <a><code>name</code></a>, <a><code>description</code></a>). This transparent information allows end-users to make a conscious decision before installing it.
+    </p>
+    <p>
+      User agents SHOULD provide a mechanism to remove an installed MiniApp.  It is RECOMMENDED that at the time of removal, the user agent also presents the user with an opportunity to revoke other persistent data and settings associated with the application, such as permissions and persistent storage.
+    </p>
+    <p>
+      The manifest's <a><code>window</code></a> member defines the appearance of the MiniApp, including the look and feel of the navigation bar and other features that modify the user agent's native UI, like <a><code>fullscreen</code></a> or <a><code>orientation</code></a>. User agents must offer intuitive mechanisms to close the apps, reverting the changes that affected the native UI. 
```

Comment:
Sorry, I do not quite understand what this paragraph has to do with privacy. It seems to have more to do with accessibility than privacy.

Reply:
Perhaps it's a bit confusing, yes. This part comes after the following question in the self-review questionnaire: 

> 11.  Do features in this specification allow an origin some measure of control over
>      a user agent's native UI?

Reply:
As suggested, this was moved to the accessibility section (3cb10f163e0785655669251a91966f4d37df77cf)